pkg/cmd/cli/describe/chaindescriber.go

```diff
@@ -92,6 +93,15 @@ func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag, includeInputImag
 		return "", NotFoundErr(fmt.Sprintf("%q", ist.Name))
 	}
 
+	markers := buildanalysis.FindCircularBuilds(g, osgraph.DefaultNamer)
+	if len(markers) > 0 {
+		for _, marker := range markers {
+			if strings.Contains(marker.Message, ist.Name) {
+				return fmt.Sprintf("Exiting examination of %s. %s", ist.Name, marker.Message), nil
```

Comment:
> can you put it in the context of the full oc status output?

So you want to display both the chain and the warning? In `oc status` we use it as a warning that users need to ask for explicitly (-v) but in the context of `oadm build-chain` I think it makes sense to print it unconditionally.